File: 프로그래머스/0/120893. 대문자와 소문자/대문자와 소문자.java
class Solution {
    public String solution(String my_string) {
        char[] ch = new char[my_string.length()];
        for(int i = 0; i < my_string.length(); i++){
            if(my_string.charAt(i) < 91 ){
                ch[i] =(char)(my_string.charAt(i)+32);
            }
            else{
                ch[i] =(char)(my_string.charAt(i)-32);
            }
        }
        String answer = String.valueOf(ch);
        return answer;
    }
}

File: 프로그래머스/0/120893. 대문자와 소문자/대문자와 소문자.cs
using System;

public class Solution {
    public string solution(string my_string) {
        char[] ch = new char[my_string.Length];
        for(int i = 0; i < my_string.Length; i++){
            if(my_string[i] < 91 ){
                ch[i] = (char)(my_string[i]+32);
            }
            else{
                ch[i] = (char)(my_string[i]-32);
            }
        }
        String answer = new String(ch);
        return answer;
    }
}